brave/browser-laptop
# New debug menu option - 'Allow Manual Tab Discarding': creates 'Discard' tab context menu command

## Test plan
- run in development mode so that the 'Debug' app menu shows
- Enable the Debug --> Allow Manual Tab Discarding option
- open some tabs
- When you right-click a tab, the 'discard' menu should show
- Click the 'discard' option in the context menu of a previously-loaded tab
- The tab contents should be discarded

Note: that you will not be able to switch to the tab due to #10673 which this debug menu option is designed to help fix!


## Submitter Checklist:


## Reviewer Checklist:


Tests
